src/components/Dashboard/Alert/AlertsBody.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { AlertItem } from './AlertItem';
+
+export const AlertsBody = ({ alerts }) => (
+  <div className="kubevirt-alert__alerts-body">
+    {alerts.map((alert, index) => (
```

Comment:
can we render the alerts as children (AlertItem)  for better reusability?

Reply:
done